MuxMediaPlayer/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.android.library'
 android {
     compileSdkVersion 28
     defaultConfig {
-        minSdkVersion 17
+        minSdkVersion 21
```

Comment:
Is there a reason for this bump? We do have people using ExoPlayer on older versions, so I'd think it would be good to keep this 16 or 17 if possible.

Reply:
@jsanford8 Correction, I think this is required in order to get access to the frame-rate, bitrate, and currently-selected video-track.

After setting to `minSdkVersion` to 17 I got:
```
  The first 3 errors (out of 5) were:
  /Users/scottkidder/git/mux-stats-sdk-mediaplayer/MuxMediaPlayer/src/main/java/com/mux/stats/sdk/muxstats/mediaplayer/MuxStatsMediaPlayer.java:234: Error: Call requires API level 21 (current min is 17): android.media.MediaPlayer#getSelectedTrack [NewApi]
          .getSelectedTrack(TrackInfo.MEDIA_TRACK_TYPE_VIDEO)];
           ~~~~~~~~~~~~~~~~
  /Users/scottkidder/git/mux-stats-sdk-mediaplayer/MuxMediaPlayer/src/main/java/com/mux/stats/sdk/muxstats/mediaplayer/MuxStatsMediaPlayer.java:235: Error: Call requires API level 19 (current min is 17): android.media.MediaPlayer.TrackInfo#getFormat [NewApi]
      if (videoTrackInfo.getFormat().containsKey(MediaFormat.KEY_FRAME_RATE)) {
                         ~~~~~~~~~
  /Users/scottkidder/git/mux-stats-sdk-mediaplayer/MuxMediaPlayer/src/main/java/com/mux/stats/sdk/muxstats/mediaplayer/MuxStatsMediaPlayer.java:236: Error: Call requires API level 19 (current min is 17): android.media.MediaPlayer.TrackInfo#getFormat [NewApi]
        sourceAdvertisedFramerate = videoTrackInfo.getFormat().getInteger(MediaFormat.KEY_FRAME_RATE);
                                                   ~~~~~~~~~
```

And after bumping to `19`, I got:
```
  Errors found:

  /Users/scottkidder/git/mux-stats-sdk-mediaplayer/MuxMediaPlayer/src/main/java/com/mux/stats/sdk/muxstats/mediaplayer/MuxStatsMediaPlayer.java:234: Error: Call requires API level 21 (current min is 19): android.media.MediaPlayer#getSelectedTrack [NewApi]
          .getSelectedTrack(TrackInfo.MEDIA_TRACK_TYPE_VIDEO)];
           ~~~~~~~~~~~~~~~~
```